src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -161,6 +162,7 @@ static const std::vector<void (*)()> charm_init_node_funcs{
     &Parallel::register_derived_classes_with_charm<MathFunction<1>>,
     &Parallel::register_derived_classes_with_charm<
         StepChooser<metavariables::step_choosers>>,
+    &Parallel::register_derived_classes_with_charm<StepController>,
```

Comment:
@wthrowe should this be removed from EvolveBurgers now that LTS is disabled there?

Reply:
The StepChooser and StepController registrations are unnecessary without LTS.  They should also be harmless, except for a (hopefully small?) increase in compile time.  In general we need to figure out a better way to manage this section.

Probably remove them for now to keep things clean.

Reply:
Okay, thanks for the pointers, @wthrowe!